test_tc10_download_cv_success.py

```diff
@@ -0,0 +1,43 @@
+import time
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.chrome.options import Options
+
+def test_download_cv():
+    chrome_options = Options()
+    chrome_options.add_argument('--no-sandbox')
+
+    # Set path to chromedriver as per your configuration
+    webdriver_service = Service("C:\\Program Files (x86)\\Google\\Chrome\\Application\\chromedriver.exe")
+    browser = webdriver.Chrome(service=webdriver_service, options=chrome_options)
+
+    # Wait for initialize, in seconds
+    wait = WebDriverWait(browser, 12)
+
+    # Get page
+    browser.get("http://cv-gen-dipl.herokuapp.com:80")
+
+    # insert unique code
+    browser.find_element(By.ID, "input-unique-code").send_keys("W6FK-ZRM5-H20C")
+    browser.find_element(By.ID, "edit-cv").click()
+    time.sleep(5)
+
+    # to next page
+    browser.find_element(By.ID, "btn-choose-template").click()
+
+    # select template
+    wait.until(EC.presence_of_element_located((By.ID, "template-formal")))
+    browser.find_element(By.ID, "template-formal").click()
+    time.sleep(1)
+    browser.find_element(By.ID, "get-cv").click()
+
+    wait.until(EC.alert_is_present())
+    browser.switch_to.alert.accept()
+
+    wait.until(EC.presence_of_element_located((By.ID, "btn-download-cv")))
```

Comment:
Jadi btn-download-cv ini pasti sudah ada di pagenya, yg membedakan siap didownload atau tidak itu bisa dari 2 hal,
1. dari tulisan "PLEASE WAIT" dan "DOWNLOAD YOUR CV"
2. dari status buttonnya, ketika disabled artinya tidak ready to download

sebenarnya kalau alert ready to downloadnya bener2 muncul gk bakal masalah sih wwkwk. Pertama kali generate CV emg lama, soalnya API Generatornya itu blom running wkwk (biasanya masih sleep kalau gk dipake dlm kurun tertentu, maklum gratisan web servicenya)

Reply:
tapi kurang lebih tidak ada problem kan yak?

Reply:
nnti w coba test